mlrun/__main__.py

```diff
@@ -1450,5 +1462,41 @@ def func_url_to_runtime(func_url, ensure_project: bool = False):
     return runtime
 
 
+def load_notification(notifications, project):
+    # A dictionary or json file containing notification dictionaries can be used by the user to set notifications.
+    # Each notification is stored in a tuple called notifications.
+    # The code then goes through each value in the notifications tuple and check
+    # if the notification starts with "file=", such as "file=notification.json," in those cases it loads the
+    # notification.json file and uses add_notification_to_project to add the notifications from the file to
+    # the project. If not, it adds the notification dictionary to the project.
+    for notification in notifications:
+        if notification.startswith("file="):
+            file_path = notification.split("=")[-1]
+            with open(file_path) as fp:
+                notification_from_file = simplejson.load(fp)
+                add_notification_to_project(notification_from_file, project)
+
+        else:
+            notification = simplejson.loads(notification)
+            add_notification_to_project(notification, project)
+
+
+def add_notification_to_project(notification, proj):
```

Comment:
```suggestion
def add_notification_to_project(notification, project):
```

also add type hints here, and every function you added on this PR